src/components/elements/PListItem.js

```diff
@@ -1,6 +1,10 @@
-import React from 'react'
+import React from "react";
 export default (props) => {
-  return (
-    <p onClick={(e) => props.onPress && props.onPress(e)}>{props.children}</p>
-  )
-}
+    const regExpHighLightKeyword = () => {
+        const regTerm = props.terms.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') // Replace special character
+        return {__html: props.children.replace(new RegExp('(' + regTerm + ')', 'i'), '<span class="marked">$1</span>')};
```

Comment:
`children` can either be text, a component or list of the components. Using it without checking the format can lead to crashes